### PR TITLE
Units string

### PR DESCRIPTION
**Pull request by Cananito**
@mmorey Review.

Addresses issue https://github.com/chaione/CHCircleGauge/issues/20

Works great for big circle gauges:

ios simulator screen shot jul 30 2014 4 57 42 pm

Not so much for small ones:

ios simulator screen shot jul 30 2014 4 58 56 pm

### CODE REVIEW

**Comment by mmorey** [comment]
But the user can change font size to make it look better, right?

File: CHCircleGauge/CHCircleGauge.h [hunk 1]
```diff
@@ -54,6 +54,11 @@ typedef NS_ENUM(NSInteger, CHCircleGaugeState) {
  */
 - (void)setValue:(CGFloat)value animated:(BOOL)animated;
 
+/**
+ String that is suffixed to the value. This string is meant to be just a few characters long. Defaults to `nil`.
+ */
+@property (nonatomic, copy) NSString *unitsString;
+
```
Comment (mmorey) [inline CHCircleGauge/CHCircleGauge.h]: So there are 2 sections:
- `/** @name Managing the gauge */`
- `/** @name Configuring the gauge */`

Would this be a better fit in the `/** @name Configuring the gauge */` section with `notApplicableString` and `noDataString`?

Comment (Cananito) [inline CHCircleGauge/CHCircleGauge.h]: Yes.

**Comment by Cananito** [comment]
Re https://github.com/chaione/CHCircleGauge/pull/28#issuecomment-50698369

True. Updating docs and dropping auto-shrinking.

**Pull request merged**